code/modules/mob/new_player/preferences_setup.dm

```diff
@@ -254,10 +254,6 @@ datum/preferences
 			facial_s.Blend(rgb(r_facial, g_facial, b_facial), ICON_ADD)
 			eyes_s.Blend(facial_s, ICON_OVERLAY)
 
-		//Slime People Overlays
-		if(current_species && (current_species.bloodflags & BLOOD_SLIME))
-			var/icon/temp = new/icon("icon" = 'icons/effects/slimemutant.dmi', "icon_state" = "[slime_color]_slime[fat]_[gender]_s")
-			preview_icon.Blend(temp, ICON_OVERLAY)

```

Comment:
Datacore.dm will need to be updated as well, it used this code, if I remember correctly.